src/sinks/postgres/config.rs

```diff
@@ -0,0 +1,147 @@
+use futures::FutureExt;
+use tower::ServiceBuilder;
+use vector_lib::{
+    config::AcknowledgementsConfig,
+    configurable::{component::GenerateConfig, configurable_component},
+    sink::VectorSink,
+};
+
+use super::{
+    service::{PostgresRetryLogic, PostgresService},
+    sink::PostgresSink,
+};
+use sqlx::{postgres::PgPoolOptions, Pool, Postgres};
+
+use crate::{
+    config::{Input, SinkConfig, SinkContext},
+    sinks::{
+        util::{
+            BatchConfig, RealtimeSizeBasedDefaultBatchSettings, ServiceBuilderExt,
+            TowerRequestConfig, UriSerde,
+        },
+        Healthcheck,
+    },
+};
+
+const fn default_pool_size() -> u32 {
+    5
+}
+
+/// Configuration for the `postgres` sink.
+#[configurable_component(sink("postgres", "Deliver log data to a PostgreSQL database."))]
```

Comment:
should I call this sink `postgres` or `postgres_logs`?

Reply:
I'm thinking if this can evolve to integrate with other postgres flavours such as timescaledb https://docs.timescale.com/use-timescale/latest/metrics-logging/